torch_geometric/explain/metrics.py

```diff
@@ -0,0 +1,69 @@
+from typing import Tuple
+
+import torch
+from torch import Tensor, tensor
+from torchmetrics import AUROC, ROC
+
+from torch_geometric.explain import Explanation
+
+
+def get_groundtruth_metrics(
+        explanation: Explanation, groundtruth: Explanation
+) -> Tuple[Tensor, Tensor, Tensor, Tensor, Tensor]:
+    """Computes different accuracy metrics when groundtruth is available.
+
+    .. note::
+        Currently we perform hard thresholding (where the threshold value
+        is set to 0) on explanation and groundtruth masks to get true
+        positives, true negatives, false positives and false negatives.
+        I.e., all values in explanation masks and ground truth masks which
+        are greater than 0 is set to 1.
```

Comment:
I think it might actually be better to have an additional argument, `threshold=0.0` which defaults to 0.0, but can be set by the user to be the thresholding of both masks. This docstring will then describe the default behavior.